docs/source/torch.rst

```diff
@@ -576,7 +576,6 @@ BLAS and LAPACK Operations
     ormqr
     outer
     pinverse
-    qr
```

Comment:
Not for this PR, but I wonder if we should leave old functions in the documentation with a very clear notice about when they were removed?

cc @stas00, who's been thinking about how we could improve the clarity of our documentation when making operator changes

Reply:
I agree having a list of removed functions somewhere in the documentation would be useful.

Reply:
yes, please! thank you for remembering my request, Mike!

Probably either making a new doc that lists all the APIs that were removed and the first pytorch version (not sha!) that no longer had it. or keeping the old doc entry with just the API name and when it was removed. Whatever is easier to maintain.

thank you!

Reply:
I like the idea of a separate doc for removed operators. What do you think, @IvanYashchuk, @Lezcano, @albanD? We could just write to it directly with the name of the operator and when it was removed.

Reply:
Makes sense to me. I think this would be a really helpful feature for libraries that depend on PyTorch

Reply:
Agreed a special page for this in the docs would be great. It should contain either details on what to use instead and/or link to the issue/PR with more details).